src/hivpy/experiment.py

```diff
@@ -1,10 +1,58 @@
 import os
-from datetime import date, timedelta
+from datetime import date, datetime, timedelta
 
-from .config import ExperimentConfig, OutputConfig, SimulationConfig
+import numpy as np
+import pandas as pd
+
+from .config import ExperimentConfig, LoggingConfig, SimulationConfig
+from .exceptions import SimulationException
 from .simulation import run_simulation
 
 
+class OutputHandler:
+    """
+    Handles output for the experiment.
+    Could be used to just keep track of files and handle writing/reading
+    Or could also be used to calculate summary statistics
+    Although if that is going to be elaborate, a separate statistics module
+    could be useful.
+    """
+    files: dict
+    output_dir: str
+    simulation_data: pd.DataFrame
+    summary_stats: np.ndarray
+
+    def __init__(self, output_params):
+        self.output_dir = os.path.normpath(output_params['OUTPUT_DIR'])
+        self.files = {"Raw": "raw_data.out", "Stats": "summary_statistics.out"}
+
+    def set_file_name(self, fileId, filename):
+        filepath = os.path.join(self.output_dir, filename)
+        self.files[fileId] = filepath
+
+    def _commit_data(self, fileId, data, write_mode):
+        if(not (fileId in self.files)):
```

Comment:
Personal preference:
```suggestion
        if fileId not in self.files:
```